### ISSUE TITLE
The output format issue

### ISSUE THREAD

**Issue by SilverIce** [DennisSoemers/PapyrusProfiler#3]
Hi! It seems like recent version of flamegraph.pl visualiser can not visualised logs (anymore?)

```
➜  Downloads flamegraph.pl PapyrusProfilerOutput_2.log > FlameGraph.svg --width 1800
Ignored 1315 lines with invalid format
ERROR: No stack counts found
```

Do you know which version could do that?

**Comment by DennisSoemers** [comment]
I just pulled the latest code from the `master` branch of [FlameGraph](https://github.com/brendangregg/FlameGraph) and it still works fine here. Maybe just something wrong with that one `.log` file? Did you try any others?

**Comment by SilverIce** [comment]
Yeah, I tried other logs, also I tried to reduce the amount of lines to find the 'bad' ones. Originally the log file had 6293 records, but all of them reported as bad ones.

Does the record format looks correct?

```
xpmseweaponquest.OnMenuOpen;xpmseweaponquest.CheckInit;xpmseweaponquest.Refresh;xpmseweaponquest.CheckCloakFlag;xpmseweaponquest.ReadXPMSEData;NiOverride.GetBodyMorph 1
nl_cmd.OnKeyDown 1
nl_cmd.OnKeyDown;nl_cmd._CON 1
nl_cmd.OnKeyDown;StringUtil.Find 1
nl_cmd.OnMenuClose;nl_cmd.UnregisterForAllKeys 1
xpmseweaponquest.OnMenuClose;xpmseweaponquest.RACESEX_MENU 1
xpmseweaponquest.OnMenuClose;xpmseweaponquest.CONSOLE_MENU 1
xpmseweaponquest.OnMenuClose;xpmseweaponquest.CheckInit;Actor.HasSpell 1
xpmseweaponquest.OnMenuClose;xpmseweaponquest.CheckInit;ActorBase.GetSex 1
```

**Comment by DennisSoemers** [comment]
@SilverIce 

Yep, just tested locally by putting the text you gave me in a `Test.log` file, and I got a proper flamegraph as output: 
![Test](https://github.com/DennisSoemers/PapyrusProfiler/assets/5389687/7d891f94-0b27-45f6-8bfd-8da2a1c0f009)

I guess I'd recommend you to just remove your local copy of FlameGraph and re-download it from scratch, to make sure you get exactly all the code that is currently online there, from [its GitHub repo](https://github.com/brendangregg/FlameGraph).

**Comment by spaceymonk** [comment]
@SilverIce Check your Line Endings! FlameGraph expects LF endings, if you are using WSL to run Perl script, it will produce this error. Log file is in CRLF format. You can try `dos2unix` to convert line endings:

```shell
dos2unix <LOG_FILE>
```
